lms/djangoapps/grades/scores.py

```diff
@@ -108,20 +108,23 @@ def get_score(submissions_scores, csm_scores, persisted_block, block):
         _get_score_from_persisted_or_latest_block(persisted_block, block, weight)
     )
 
-    assert weighted_possible is not None
-    has_valid_denominator = weighted_possible > 0.0
-    graded = _get_graded_from_block(persisted_block, block) if has_valid_denominator else False
-
-    return ProblemScore(
-        raw_earned,
-        raw_possible,
-        weighted_earned,
-        weighted_possible,
-        weight,
-        graded,
-        display_name=display_name_with_default_escaped(block),
-        module_id=block.location,
-    )
+    if weighted_possible is None or weighted_earned is None:
+        return None
```

Comment:
I know we're trying to get this done quickly for the release, but is there a follow-up PR with a regression test for this situation?